src/backend/src/utils/change-requests.utils.ts

```diff
@@ -68,3 +68,28 @@ export const validateChangeRequestAccepted = async (crId: number) => {
 
   return changeRequest;
 };
+
+/**
+ * Validates that there are no unchecked expected activities or delivrerables
+ * @param workPackage Work package to check bullets for
+ * @throws if there are any unchecked expected activities or deliverables
+ */
+export const throwIfUncheckedDescriptionBullets = (
+  workPackage: (Work_Package & { expectedActivities: Description_Bullet[]; deliverables: Description_Bullet[] }) | null
+) => {
+  // if it's a work package, all deliverables and expected activities must be checked
+  if (workPackage) {
+    const wpExpectedActivities = workPackage.expectedActivities;
```

Comment:
`const { expectedActivities, deliverables } = workPackage;`

no need to rename especially with `wp` which i want to avoid everywhere possible